Sources/ArcGISToolkit/Utility/AsyncImageView.swift

```diff
@@ -94,8 +94,7 @@ public struct AsyncImageView: View {
                 }
                 .padding([.top, .bottom])
             }
-            if #available(iOS 16.0, macCatalyst 16.0, *),
-               let progressInterval = viewModel.progressInterval {
+            if let progressInterval = viewModel.progressInterval {
 #if canImport(Charts)
```

Comment:
You can remove all instances of `#if canImport(Charts)` from the code. `Charts` is only available on iOS 16; now that it's the minimum, everyone gets Charts!